crates/sui-storage/src/indexes.rs

```diff
@@ -878,12 +878,12 @@ impl IndexStore {
         // We use +1 to grab the next cursor
         let count = min(count, MAX_GET_OWNED_OBJECT_SIZE + 1);
         debug!(?owner, ?count, ?starting_object_id, "get_owner_objects");
-        Ok(self
+        let iter = self
             .tables
             .owner_index
             .iter()
             // The object id 0 is the smallest possible
-            .skip_to(&(owner, starting_object_id))?
+            .skip_to(&(owner.clone(), starting_object_id))?
```

Comment:
for impl. of exclusive cursor, you can refer codes [here](https://github.com/MystenLabs/sui/pull/9017/files#diff-5256c3821e83e774a50c788b7d35793b09728f20bb1a8d17c55f68c700dc20a7)